src/home/pages/Home.vue

```diff
@@ -80,108 +124,242 @@
   {
-    image: '',
-    title: 'Sermon 5'
-  }, {
-    image: '',
-    title: 'Sermon 6'
+    image: "",
+    title: "",
+  }
+])
+let sermonImgs = [String]
+const sermonCar = ref([]);
+
+onMounted(async () => {
+  var htmlVerse = await getVerseOfTheDay();
+
+  verseName.value = htmlVerse.verseReference + " KJV";
+  verseText.value = htmlVerse.verseText;
+
+  const recentDevotionalsPromise = getRecentDevotionals(1, 6);
+  const recentSermonsPromise = getRecentSermons(1, 6);
+  const [recentDevotionals, recentSermons] = await Promise.all([recentDevotionalsPromise, recentSermonsPromise]);
+
+  for (let i = 0; i < devoList.value.length; i++) {
+    if(recentDevotionals[i].coverImage == ""){
+      devoImgs[i] = recentDevotionals[i].author?.profileImage;
+    }
+    else{
+      devoImgs[i] = getBucketUrl(recentDevotionals[i], recentDevotionals[i].coverImage);
+    }
   }
-]
 
-const { breakpoint } = useBreakpoint()
+  devoList.value = [
+    {
+      image: devoImgs[0],
+      title: recentDevotionals[0].title,
+      link: `devotionals/${recentDevotionals[0].id}`
+    },
+    {
+      image: devoImgs[1],
+      title: recentDevotionals[1].title,
+      link: `devotionals/${recentDevotionals[1].id}`
+    },
+    {
+      image: devoImgs[2],
+      title: recentDevotionals[2].title,
+      link: `devotionals/${recentDevotionals[2].id}`
+    },
+    {
+      image: devoImgs[3],
+      title: recentDevotionals[3].title,
+      link: `devotionals/${recentDevotionals[3].id}`
+    },
+    {
+      image: devoImgs[4],
+      title: recentDevotionals[4].title,
+      link: `devotionals/${recentDevotionals[4].id}`
+    },
+    {
+      image: devoImgs[5],
+      title: recentDevotionals[5].title,
+      link: `devotionals/${recentDevotionals[5].id}`
+    },
+  ];
+
+  for (let i = 0; i < sermonList.value.length; i++) {
+    if(recentSermons[i].coverImage == ""){
+      sermonImgs[i] = recentSermons[i].author?.profileImage;
+    }
+    else{
+      sermonImgs[i] = recentSermons[i].coverImage
+    }
+  }
 
-const router = useRouter()
+
+  sermonList.value = [
+    {
+      image: sermonImgs[0],
+      title: recentSermons[0].title,
+      link: `sermons/${recentSermons[0].id}`
+    },
+    {
+      image: sermonImgs[1],
+      title: recentSermons[1].title,
+      link: `sermons/${recentSermons[1].id}`
+    },
+    {
+      image: sermonImgs[2],
+      title: recentSermons[2].title,
+      link: `sermons/${recentSermons[2].id}`
+    },
+    {
+      image: sermonImgs[3],
+      title: recentSermons[3].title,
+      link: `sermons/${recentSermons[3].id}`
+    },
+    {
+      image: sermonImgs[4],
+      title: recentSermons[4].title,
+      link: `sermons/${recentSermons[4].id}`
+    },
+    {
+      image: sermonImgs[5],
+      title: recentSermons[5].title,
+      link: `sermons/${recentSermons[5].id}`
+    },
+  ];
+});
+
+watch(devoList, () => {
+  //@ts-expect-error
+  devoCar.value = devoList;
+})
+
+watch(sermonList, () => {
+  //@ts-expect-error
```

Comment:
What error is being thrown here, we shouldn't get in the habit of ignoring errors from typescript